src/dali_backend.h

```diff
@@ -164,20 +173,34 @@ AllocateOutputs(
   return ret;
 }
 
-void
+
+struct RequestMeta {
+  uint64_t compute_start_ns, compute_end_ns;
+  int batch_size;
+};
+
+
+RequestMeta
 ProcessRequest(
```

Comment:
Why is this function in the header?

Reply:
I've put the dali_backend-specific functions inside a header to have clearer separation of the custom routines and backend's boiler-plate.